apps/dapp-console-api/src/utils/faucetBalances.ts

```diff
@@ -0,0 +1,91 @@
+import type { Address } from 'viem'
+
+import type { RedisCache } from './redis'
+
+const getFaucetContractBalanceKey = (address: Address, chainId: number) =>
+  `faucet_contract_balance_${chainId}_${address}`
+
+export const updateFaucetContractBalance = async ({
```

Comment:
we wont be doing any balance updates from dapp-console-api, right? If I remember correctly, gateway is handling the updates and dapp-console is just reading from the cache. this and all other updates can be removed if that is the case

Reply:
Should we hold off on deleting on things related to the poller and cache since we want to eventually migrate that over? Open to either direction

Reply:
In general, a good rule of thumb is to to delete things that arent being used, that way you dont end up having to maintain code that is unused. Also, if we end up using it later, it will be straightforward to add it back.